examples/hono/src/deno.ts

```diff
@@ -0,0 +1,23 @@
+import { Hono } from 'hono'
+import { upgradeWebSocket } from 'hono/deno'
+import { newHttpBatchRpcResponse, newWebSocketRpcSession } from '../../../dist/index.js'
+import { MyApiServer } from './my-api-server.ts'
+
+const app = new Hono()
+
+app.get(
+  '/api',
+  upgradeWebSocket((_c) => {
+    return {
+      onOpen(_event, ws) {
+        newWebSocketRpcSession(ws.raw!, new MyApiServer())
```

Comment:
I imagine this doesn't work on Bun, since it doesn't use the standard `WebSocket` API under the hood. Is that right?

I wonder if it makes sense to write a custom transport specifically for Hono, that uses Hono's wrapper and so works with all runtimes Hono supports?

Although I see Hono's `upgradeWebSocket` is actually different for each runtime, so perhaps this code can't really be totally runtime-independent anyway?

What do you think?

Reply:
there are plenty of hono-* packages, so hono-capnweb can totally be a thing (or even officially as a `@hono/` package

Reply:
Ah, there is a ready #61! [The approch](https://github.com/cloudflare/capnweb/issues/61#issuecomment-3404005663) by @grampelberg is great. With it, we can make Cap'n Web support Bun independent of Hono. Let's discuss it on #61.

We, Hono side, will develop "Cap'n Web Middleware" as `@hono/capnweb` in our repo ([honojs/middleware](https://github.com/honojs/middleware)), not in this repo.

Regarding this PR, I'll revise after creating `@hono/capnweb`.